.circleci/config.yml

```
@@ -0,0 +1,89 @@
+      - run:
+          name: Install cmake
+          no_output_timeout: 60m
+          command: |
+              wget https://github.com/Kitware/CMake/releases/download/v3.13.4/cmake-3.13.4-Linux-x86_64.sh
+              sudo mkdir /opt/cmake
+              sudo sh ./cmake-3.13.4-Linux-x86_64.sh --prefix=/opt/cmake --skip-license
+              sudo ln -s /opt/cmake/bin/cmake /usr/local/bin/cmake
+      - run:
+          name: Install conda
+          no_output_timeout: 60m
+          command: |
+              curl -o ~/miniconda.sh -O  https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+              chmod +x ~/miniconda.sh
+              ~/miniconda.sh -b -p $HOME/miniconda
+              rm ~/miniconda.sh
+              $HOME/miniconda/bin/conda install -y numpy pyyaml scipy ipython mkl mkl-include
+              export PATH=$HOME/miniconda/bin:$PATH
+              conda create -y -n habitat python=3.6
+              . activate habitat
+              conda install -y numpy pyyaml scipy ipython mkl mkl-include pytest
+              sudo apt-get update || true
+              sudo apt-get install -y --no-install-recommends \
+                  build-essential \
+                  git \
+                  curl \
+                  vim \
+                  ca-certificates \
+                  libjpeg-dev \
+                  libglm-dev \
+                  libegl1-mesa-dev \
+                  xorg-dev \
+                  freeglut3-dev \
+                  pkg-config \
+                  wget \
+                  zip \
+                  unzip || true
+      - run:
+          name: Install cuda
+          no_output_timeout: 60m
+          command: |
+              wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/cuda-repo-ubuntu1604_8.0.44-1_amd64.deb
+              sudo dpkg -i cuda-repo-ubuntu1604_8.0.44-1_amd64.deb
+              sudo apt-get update || true
+              sudo apt-get --yes --force-yes install cuda
+              nvidia-smi
+      - run:
+          name: Install habtiat-sim
+          no_output_timeout: 60m
+          command: |
+              export PATH=$HOME/miniconda/bin:$PATH
+              . activate habitat; cd habitat-sim
+              python setup.py install --headless
+      - run:
+          name: Download test data
+          command: |
+              cd habitat-sim
+              wget http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip
+              unzip habitat-test-scenes.zip
+              rm habitat-test-scenes.zip
+      - run:
+          name: Run sim tests
+          command: |
+              cd habitat-sim
+              export PATH=$HOME/miniconda/bin:$PATH
+              . activate habitat
+              python examples/example.py --scene data/scene_datasets/habitat-test-scenes/van-gogh-room.glb
+              export PYTHONPATH=$PYTHONPATH:$(pwd)
+              ./build.sh --headless
+              pip install -e .
```

Possibly bad things will happen as the pip install isn't headless.

```suggestion
              HEADLESS=True pip install -e .
```